package.json

```diff
@@ -4,7 +4,10 @@
   "private": true,
   "description": "EduLaw is a project that assists parents in reporting violations of their children's educational rights to the appropriate legal entity.",
   "scripts": {
-    "dev": "next dev",
+    "dev": "npm run db:start && next dev",
```

Comment:
These scripts allow for database creation to be baked into our normal commands:
- `npm run dev` - starts both the database and the localhost server
- `npm run stop`, `npm run db:stop` - stops the database
- `npm run db:start` - starts the database but not the localhost server